.changeset/serious-rules-grow.md

```diff
@@ -0,0 +1,5 @@
+---
+"@digdir/designsystemet-react": patch
+---
+
+**Suggestion**: Set tabindex="-1" on list, fixes keyboard focus trap
```

Comment:
Are these changesets correct now?

Reply:
Well they are not wrong, but it's not something we do directly anymore

Reply:
Maybe just the "fixes keyboard focus trap" since we didn't fix directly but its fixed by updating a dependency.